lib/wikidata/diff/analyzer.rb

```diff
@@ -177,8 +302,8 @@ def self.get_revision_content(revision_id)
     end
   end
 
-  # Gets the parent id based on current revision id from the action:compare at Wikidata API.
-  def self.get_parent_id(current_revision_id)
+   # Gets the parent id based on current revision id from the action:compare at Wikidata API.
+   def self.get_parent_id(current_revision_id)
```

Comment:
Is there a way to get the parent ID for more than one revision at a time? Making this query for every revision will be a significant bottleneck.